packages/strapi-plugin-upload/admin/src/components/UploadList/index.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { ModalBody } from 'strapi-helper-plugin';
+import { Button } from '@buffetjs/core';
+import createMatrix from '../../utils/createMatrix';
+import getTrad from '../../utils/getTrad';
+import CardImgWrapper from '../CardImgWrapper';
+import InfiniteLoadingIndicator from '../InfiniteLoadingIndicator';
+import HeaderWrapper from './HeaderWrapper';
+
+const UploadList = ({
+  filesToUpload,
+  onClickCancelUpload,
+  onGoToAddBrowseFiles,
+}) => {
+  const matrix = createMatrix(filesToUpload);
+  const filesToUploadLength = filesToUpload.length;
+  const titleId = `modal.upload-list.sub-header-title.${
+    filesToUploadLength > 1 ? 'plural' : 'singular'
+  }`;
+
+  return (
+    <>
+      <HeaderWrapper>
+        <div>
+          <div className="assets-selected">
+            <FormattedMessage
+              id={getTrad(titleId)}
+              values={{ number: filesToUploadLength }}
+            />
+          </div>
+          <div className="infos">
+            <FormattedMessage
+              id={getTrad('modal.upload-list.sub-header-subtitle')}
+              values={{ number: filesToUploadLength }}
+            />
+          </div>
+        </div>
+        <div>
+          <FormattedMessage id={getTrad('modal.upload-list.sub-header.button')}>
+            {label => (
+              <Button
+                type="button"
+                color="primary"
+                label={label}
+                onClick={onGoToAddBrowseFiles}
+              />
+            )}
+          </FormattedMessage>
+        </div>
+      </HeaderWrapper>
+      <ModalBody>
+        <div
+          className="container"
+          style={{
+            marginTop: -4,
```

Comment:
Don't forget to avoid inline style. The negative margin is a consequence of a bad wrapper (ModalBody) style. This is not scalable.

Reply:
In this particular case not really it is more because the design is not the same for all modals.